cwm_worker_operator/updater.py

```diff
@@ -17,6 +17,7 @@
 
 
 DATETIME_FORMAT = '%Y%m%dT%H%M%S%z'
+CONFIRM_FORCE_DELETE_DATA = 'Yes, confirm delete of data!'
```

Comment:
The word `deletion` instead of `delete` would make more sense.

```python
CONFIRM_FORCE_DELETE_DATA = 'Yes, confirm deletion of data!'
```

From the rest of the code, I gathered that it is just being used as a flag.
It could be anything, right?

Reply:
yes, it's just a placeholder text to make it slightly harder to set this redis key by mistake..